lnwallet/reservation.go

```diff
@@ -221,6 +238,23 @@ func NewChannelReservation(capacity, fundingAmt, feePerKw btcutil.Amount,
 		chanOpen:      make(chan *openChanDetails, 1),
 		chanOpenErr:   make(chan error, 1),
 		wallet:        wallet,
+		RequiredRemoteChanReserve: func(cap btcutil.Amount) btcutil.Amount {
```

Comment:
Note that `cap` is a reserved key-word. As a result, I suggest using `capacity` here instead.

Reply:
Also I dig allowing these parameters to be configurable, but IMO this should instead be lifted up the the funding manager. As that's where we have similar closures such as the one used to determine what the CSV value for a given channel size should be.